_guides/themes.md

```diff
@@ -33,21 +33,19 @@ TypeDoc ships with two themes so you can start creating docs right out of the bo
 
 ## Custom themes
 
-Creating a custom theme in TypeDoc is pretty easy. Basically you create a directory and overwrite the
-components of the [``default`` theme](https://github.com/TypeStrong/typedoc-default-themes/tree/master/src/default) 
-you want to change. All themes extend the default theme, so you only have to change the parts that don't 
-fit your needs.
+Creating a custom theme in TypeDoc is pretty easy. Basically you copy the  [default theme directory](https://github.com/TypeStrong/typedoc-default-themes/tree/master/src/default) (from which all themes extend from) and update the parts you want to change.
 
-This is the basic directory structure of a theme, only the assets directory is mandatory:
+Then simply run `--themes` and pass it the path to the directory
 
+```bash
+$ typedoc --out path/to/documentation/ --theme path/to/custom-theme path/to/typescript/project/
 ```
-+ my-theme             // Root directory of your theme
-  + assets             // Static assets like *.css or *.js files
-  + layouts            // Optional. Layout templates
-    - default.hbs      // Optional. Default layout template
-  + partials           // Optional. Partials to be used by Handlebar
-  + templates          // Optional. Page templates
-    - index.hbs        // Optional. Home page template
-    - reflection.hbs   // Optional. Definition page template
-  - theme.js           // Optional. JavaScript class of the theme
-```
+
```

Comment:
I think I'd prefer to leave the directory structure to explain what the different pieces are. It already indicates that most parts are optional.

Reply:
@aciccarello  sounds good, I'd be happy to add that in. I did notice that the old docs had "assets directory is mandatory" but I definitely ran it without including the assets directory so I'm going to leave that part out unless you can think of some reason it might be that I'm not considering.